CONTRIBUTING.md

```diff
@@ -1,48 +1,79 @@
 # Contributing to Fathom
 
+## Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [Matrix room](#matrix-room)
+- [Prerequisites](#prerequisites)
+- [Workflow](#workflow)
+- [Quality standards](#workflow)
+- [Background](#background)
+
 ## Code of Conduct
 
 Please note that this project is released with a [Code of Conduct](./CODE_OF_CONDUCT.md).
 By participating in this project you agree to abide by its terms.
 
-## Join the matrix room
+## Matrix room
 
-Joining the matrix room at [#fathom-lang:matrix.org](https://app.element.io/#/room/#fathom-lang:matrix.org) is a good way to get in touch with the developers and community.
+Joining the matrix room at [#fathom-lang:matrix.org][fathom-matrix] is a good way to get in touch with the developers and community.
 
-## Getting started with development
+[fathom-matrix]: https://app.element.io/#/room/#fathom-lang:matrix.org
 
-### Install Rust
+## Prerequisites
 
-We use [Rust](https://www.rust-lang.org/) as our implementation language.
-You can learn more about programming in Rust by reading
-[The Rust Programming Language](https://doc.rust-lang.org/book/).
+We use [Rust][rust] as our implementation language, which can be installed using the [rustup] tool.
 
-For the best experience in working with Rust, you'll want to also install IDE
-support for your editor of choice:
+Once rustup is installed we recommend using it to install [rustfmt]:
 
-- [Rust Analyzer](https://rust-analyzer.github.io/) (for VS Code, Vim Emacs, etc.)
-- [IntelliJ Rust](https://intellij-rust.github.io/) (for IntelliJ-based IDEs)
+```sh
+rustup component add rustfmt
+```
```

Comment:
I think this section can be removed now. `rustfmt` is installed by default (just double checked by running an install in a Docker container).

Reply:
Oh cool, didn't realise that!